specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```diff
@@ -300,6 +300,78 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{resourceName}/listresourcepreparationerrors": {
+      "get": {
+        "tags": [
+          "ProxyOperations"
+        ],
+        "operationId": "ResourcePreparationErrors_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
```

Comment:
Well after this change, we will have to rename the path variables in the URI accordingly, for instance the `workspaceNameParameter` requires the corresponding path variable with name `workspaceName`.
This is breaking the semantic CI checks.

Reply:
Cool, will change it